src/components/SideBar.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { FaTimes } from "react-icons/fa";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+const Sidebar = (props) => {
```

Comment:
Kindly leave a single line space between module imports and function declaration.